templates/server-configmap.yaml

```diff
@@ -39,11 +39,17 @@ data:
             {{- toYaml . | nindent 12 }}
             {{- end }}
           {{- end }}
-          {{- if $.Values.server.config.persistence.faultinjection}}
-          {{- if $.Values.server.config.persistence.faultinjection.rate }}
+          {{- if or $.Values.server.config.persistence.default.faultInjection $.Values.server.config.persistence.faultInjection}}
```

Comment:
Old simple "just `rate`" format is still supported to keep backward compatibility.

Reply:
We may not need this backward compatibility.
I'd go with a simple version without backward compatibility.